feed.reports.xml

```diff
@@ -0,0 +1,31 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+    <channel>
+        <title>{{ site.title | xml_escape }}</title>
+        <description>{{ site.description | xml_escape }}</description>
+        <link>{{ site.url }}{{ site.baseurl }}/</link>
+        <atom:link href="{{ '/feed.reports.xml' | prepend: site.baseurl | prepend: site.url }}" rel="self" type="application/rss+xml"/>
+        <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
+        <lastBuildDate>{{ site.time | date_to_rfc822 }}</lastBuildDate>
+        <generator>Jekyll v{{ jekyll.version }}</generator>
+        {% for report in site.reports %}
+            <item>
+                <title>{{ report.title | xml_escape }}</title>
+                <author>{{ report.author | xml_escape }} and the Teach LA community</author>
+                <description>{{ report.content | xml_escape }}</description>
```

Comment:
The description of each post is *really long*!

![long description](https://user-images.githubusercontent.com/7704542/107706595-6cf68200-6c75-11eb-8dc2-87b0773ec02a.png)

Instead of using the escaped content, instead you could use a [post excerpt](https://jekyllrb.com/docs/posts/#post-excerpts). You can also look into the `limit` filter to improve this experience :+1:

```suggestion
                <description>{{ report.excerpt | xml_escape }}</description>
```

Reply:
I see, sounds good! I will try to research a bit about how to add the RSS icon and how the post excerpt works (I may have to ask you some questions in slack if I come into any issues haha). 

I thought that the jekyll.feed plugin auto-generated a feed.xml for the normal blog posts, which was why I added a reports xml file to generate the posts specifically for report.

Reply:
Yeah, I think the current setup XML-wise (one for blog posts, one for reports) is totally okay. The two have different fields, which makes mixing and matching them a bit frustrating. And, as Stephanie has pointed out, `jekyll-feed` already autogenerates the `feed.xml` for the `_posts` collection, so we're good!

As for the icon - we already use [font awesome](https://fontawesome.com/) in our project, so adding an icon could be as simple as 

```html
<span class="fa fa-rss-square></span> Subscribe to the RSS Feed
```

That being said, some of this could be out of scope of the PR - feel free to tackle this in another pull request/hand it over to someone else.

Reply:
As for the full-text description; afaik many RSS feeds embed the entire content into the tag? I'm willing to do whatever `jekyll-feed` defaults to, which seems like this behaviour.